### PR TITLE
Up next for you

### PR DESCRIPTION
**Pull request by vinnyjth**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by vinnyjth** [comment]
@conrad-vanl @burkeshartsis @jordanwade  can one of you smoke test this for me and and make sure it works well with whatever content your user has consumed?

File: packages/apollos-api/src/data/content-items/data-source.js [hunk 1]
```diff
@@ -16,22 +23,6 @@ class ExtendedContentItem extends ContentItem.dataSource {
...
-
-    return cursor.orderBy('Priority'); // Changed from Core, ordering by Priority instead of 'order'
-  };
-
-  _getCursorBySiblingContentItemId = this.getCursorBySiblingContentItemId;
-
-  getCursorBySiblingContentItemId = async (id) => {
-    const cursor = await this._getCursorBySiblingContentItemId(id);
-
-    return cursor.orderBy('Priority'); // Changed from Core, ordering by Priority instead of 'order'
```
Comment (conrad-vanl) [inline packages/apollos-api/src/data/content-items/data-source.js:32]: @vinnyjth whats' the reason for taking this out?

File: packages/apollos-api/src/data/content-items/data-source.js [hunk 2]
```diff
@@ -230,6 +221,69 @@ class ExtendedContentItem extends ContentItem.dataSource {
     return [mostRecentSermon];
   };
 
+  async getSeriesWithUserProgress() {
```
Comment (conrad-vanl) [inline packages/apollos-api/src/data/content-items/data-source.js:224]: What do we have on core? How does this differ? Did this just not get built on core ever?

File: packages/apollos-api/src/data/content-items/data-source.js (hunk 1 above)
Comment (vinnyjth) [reply packages/apollos-api/src/data/content-items/data-source.js:32]: I had patched core to solve this problem - now handled here: 

https://github.com/Differential/willow-creek/pull/125/files#diff-36c37c7a05f5ea49ddc96127943f7f10R12

File: packages/apollos-api/src/data/content-items/data-source.js (hunk 2 above)
Comment (vinnyjth) [reply packages/apollos-api/src/data/content-items/data-source.js:224]: Core works slightly differently, and I think core should be updated to the logic here. I made this realization while testing, and so had to write this code. 

Core says - if I have completed 100% of a series, don't show it. 
Willow says - if I have completed the most recent entry of a series, don't show it. 

The problem is that the "Continue" button won't show up if you have completed 50% of a series, but have completed the most recent entry.

Comment (vinnyjth) [reply packages/apollos-api/src/data/content-items/data-source.js:224]: It's only that "getLastChildComplete" method call on line 255 that changes.

**Comment by vinnyjth** [comment]
@conrad-vanl if my comments make sense I'll merge this and get started on deploying it.

**Comment by conrad-vanl** [comment]
@vinnyjth merged, ship it
